src/app/containers/NewStoryPromo/PromoAtoms/Image.styles.jsx

```diff
@@ -0,0 +1,3 @@
+import styled from '@emotion/styled';
+
+export default styled.img``;
```

Comment:
General question on the `.styles.jsx` file naming convention, at the end of the day these behave as components with emotion attaching a class name to it using the styled API. does the naming convention help?

Reply:
I don't know, tried to find an answer online, but could not find anything. I believe this is just a name convention that allows us to name the style file the same way as the file where the styles are applied and help us identify where the styles are. But I am not sure if it has any technical help.